pep-0605.rst

```diff
@@ -460,48 +548,47 @@ interests of operating environments that are able to consume new releases as
 fast as the CPython core team is prepared to produce them.
 
 
-Why rolling beta releases rather than rolling alpha releases?
-----------------------------------------------------------------
-
-The code quality standards upheld by the CPython code review process and
-BuildBot fleet make the "beta" label more suitable than the "alpha" label.
-
-The "production ready for some environments, but subject to change with limited
-notice" caveat also aligns well with at least some uses of the "beta" term,
-whereas "alpha" almost universally indicates "not yet ready for production".
+Is it necessary to keep the "alpha" and "beta" naming scheme?
+-------------------------------------------------------------
 
+Using the "a" and "b" initials for the proposed rolling releases is a design
+constraint imposed by some of the pragmatic aspects of the way CPython version
+numbers are published.
 
-Why rolling beta releases rather than something like "brisk release cadence"?
------------------------------------------------------------------------------
+Specifically, alpha releases, beta releases, and release candidates are reported
+in some places using the strings "a", "b", and "c" respectively, while in others
+they're reported using the hex digits ``0xA``, ``0xB``, and ``0xC``. We want to
+preserve that, while also ensuring that any ``Python-Requires`` constraints
+are expressed against the beta releases rather than the alpha releases (since
+the latter may not enforce the ``abi3`` stability requirements if two alpha
+releases occur in succession).
 
-Using the "b" initial for the proposed rolling releases is a design constraint
-imposed by some of the pragmatic aspects of the way CPython version numbers are
-published (specifically, alpha releases, beta releases, and release candidates
-are reported in some places using the strings "a", "b", and "c" respectively,
-while in others they're reported using the hex digits ``0xA``, ``0xB``, and
-``0xC``, and we want to preserve that, while also ensuring that the rolling
-releases are all sorted after any alpha releases, and before the first release
-candidate).
-
-However, there isn't anything forcing us to say that the "b" stands for "beta".
+However, there isn't anything forcing us to say that the "a" stands for "alpha"
+or the "b" stands for "beta".
 
 That means that if we wanted to increase adoption amongst folks that were
-only being put off by the "beta" label, then it could make sense to rebrand the
-rolling beta releases as "brisk" releases, emphasising the rate of release over
-the fact that the release includes APIs and ABIs that aren't necessarily stable
-yet.
+only being put off by the "beta" label, then it may make sense to emphasise
+the "ABI breaking" and "binary compatible" names over the "alpha" and "beta"
```

Comment:
👏 Nice

Maybe emphasise the initials of the two names to make the a/b clearer?

```suggestion
the "*A*BI breaking" and "*B*inary compatible" names over the "alpha" and "beta"
```